tests/helpers/test_dispatcher.py

```diff
@@ -8,7 +9,7 @@
 from tests.common import get_test_home_assistant
 
 
-class TestHelpersDispatcher:
+class TestHelpersDispatcher(unittest.TestCase):
```

Comment:
this was tests done as a pytest test class, not as a unittest testcase. That's why it is setup_method and not unittest `setUp`. Why did you change this?

We prefer tests to be just functions btw, and no more have any classes for tests.

Reply:
Changed to be able to use `self.assertLogs` below

Reply:
Use `caplog` argument instead and pytest standalone tests.

Reply:
OK, fixed.